src/app/modules/core/components/user-profile/user-profile.component.html

```diff
@@ -1,6 +1,101 @@
-
 <div class="container-fluid">
-  <app-profile-header></app-profile-header>
-  <app-contact-list></app-contact-list>
+  <app-profile-header (screenScope)="updateScreenScope($event)"></app-profile-header>
+  <app-contact-list *ngIf="screenScope.contactList"></app-contact-list>
```

Comment:
Cant we design it like this?
1) A dashboard / home component which shows contact list
2) a separate route for user profile when user clicks on user profile button.
3) Separate out contact list from user profile component since they both are independent. 
What is your idea about that? Which one would be better?

Reply:
I agree with the above suggestion